client/core/core.go

```diff
@@ -1494,17 +1535,15 @@ func (c *Core) wait(assetID uint32, trigger func() (bool, error), action func(er
 
 func (c *Core) notifyFee(dc *dexConnection, coinID []byte) error {
 	if dc.acct.locked() {
-		return fmt.Errorf("%s account locked. cannot notify fee.", dc.acct.host)
+		return fmt.Errorf("%s account locked. cannot notify fee. log in first.", dc.acct.host)
 	}
 	// Notify the server of the fee coin once there are enough confirmations.
 	req := &msgjson.NotifyFee{
 		AccountID: dc.acct.id[:],
 		CoinID:    coinID,
 	}
-	// We'll need this to validate the server's acknowledgement.
-	reqB := req.Serialize()
-	// Sign the request and send it.
-	err := stamp(dc.acct.privKey, req)
```

Comment:
Was serializing the `NotifyFee` before stamping it, but the `Time` field is included in the serialization.  When the server signs the ack, the message is the *stamped* `NotifyFee`, so the signature verification in the response handler below must have been failing.

In any case, I'm not getting the "Account was registered, but DEX signature could not be verified.." warning below, so the `checkSig` is succeeding:

```
[INF] CORE: Attempting registration fee payment for SsoFtubsqCf57ZVqBkbzBSJZZ3XFnEPynWS of 100000000 units of dcr
[DBG] CORE[dcr]: Change output size = 36, addr = SseBhiCcFbQ84mTTimfMpHt2wbm5TFwdNrn
[DBG] CORE[dcr]: 3 signature cycles to converge on fees for tx 7e491d6e9b3c240b022b0b2963c58369b409270b21bb8d489a4a5c39a4c1d06a: min rate = 11, actual fee rate = 11 (2772 for 252 bytes), change = true
[INF] CORE: notify: |SUCCESS| (feepayment) Fee payment in progress - Waiting for 1 confirmations before trading at 127.0.0.1:7232
[DBG] CORE: Registration fee txn 7e491d6e9b3c240b022b0b2963c58369b409270b21bb8d489a4a5c39a4c1d06a:0 now has 1 confirmations.
[INF] CORE: Notifying dex 127.0.0.1:7232 of fee payment.
[DBG] CORE: Authenticated connection to 127.0.0.1:7232, 0 active matches
[INF] CORE: notify: |SUCCESS| (feepayment) Account registered - You may now trade at 127.0.0.1:7232
```